EIPS/eip-2535.md

```diff
@@ -0,0 +1,464 @@
+---
+eip: 2535
+title: Diamond Standard
+author: Nick Mudge <nick@perfectabstractions.com>
+discussions-to: https://github.com/ethereum/EIPs/issues/2535
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-02-22
+replaces: 1538
```

Comment:
One can only replace/supersede a final EIP. I suggest to:
- remove this line
- mention this somewhere in the motivation or another appropriate section
- since 1538 is also authored by you, independently of this PR mark it as "Abandoned"

Reply:
@axic Makes sense and good suggestions.   

I removed the "replaces" line and mentioned the replacement in the text of the EIP.   I updated the pull request with the changes.

I'll go to EIP 1538 and mark it as abandoned and replaced with eip 2535.

Reply:
In this pull request EIP-1538 is now marked as abandoned and replaced by EIP-2535.